vtex-apps/open-graph
# Update maintenance tooling

#### What problem is this solving?

This repo was not updated to use our latest linting and testing setups. Just a simple `vtex setup --tooling` and a few fixes made by `eslint --fix` did the trick 😄 .

#### How to test it?

There isn't much to see here. If nothing is broken by these changes, this PR does what it's supposed to.

You can go to this [Workspace](https://victormiranda--storecomponents.myvtex.com/vintage-phone/p) and check the HTML looking for `meta` tags such as these:

- Without this branch linked

<img width="1061" alt="Captura de Tela 2020-06-24 às 14 11 42" src="https://user-images.githubusercontent.com/27777263/85601970-453ea500-b625-11ea-841b-0c896abeb011.png">


- With this branch linked

<img width="1032" alt="Captura de Tela 2020-06-24 às 14 14 44" src="https://user-images.githubusercontent.com/27777263/85601888-2e984e00-b625-11ea-98ff-fcdca52c7fa3.png">


#### Screenshots or example usage:

<img width="735" alt="Captura de Tela 2020-06-24 às 14 20 02" src="https://user-images.githubusercontent.com/27777263/85602481-cc8c1880-b625-11ea-99cf-a82aa4099f01.png">


#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/d2ZeMUDQSSsCP9FC/giphy.gif)
